source/cli.js

```diff
@@ -31,6 +31,7 @@ const cli = meow(`
 	  --no-yarn           Don't use Yarn
 	  --contents          Subdirectory to publish
 	  --no-release-draft  Skips opening a GitHub release draft
+	  --test-script       User-defined script to replace default test before publish
```

Comment:
Still not that clear.
```suggestion
	  --test-script     Name of npm run script to run tests before publishing [default: test]
```